### PR TITLE
Sanitize html input before appending

### LINKED ISSUE #6: Sanitize all HTML in front-end

### PR DESCRIPTION
**Pull request by patrickhpan**
Fixes #6

### CODE REVIEW

**Comment by patrickhpan** [comment]
DO NOT APPROVE DO NOT MERGE UNTIL 10/27

File: index.html [hunk 1]
```diff
@@ -75,17 +75,26 @@
...
+                        $newDiv.text(data);
+                        $chat.append($newDiv);
                     })
                     socket.on("waiting", () => {
                         $chat.append("<em>Waiting for a partner...</em><br />")
                     })
                     socket.on("pairing", function(data) {
-                        $chat.text("")
-                        $chat.append("<em>You are now talking to " + data + "</em><br />")
+                        var $newEm = $("<em />");
```
Comment (russellpekala) [inline index.html]: please add a comment

**Review by russellpekala — APPROVED** [review]
nice

**Review by nickboucher — APPROVED** [review]
Nice jQuery Implementation!

**Pull request merged**